tests/local_db/test_data_access.py

```diff
@@ -112,7 +112,7 @@ def test_delete_file_from_request_bad_state():
     ],
 )
 def test_withdraw_file_from_request_bad_state(status, mock_old_api):
-    author = factories.create_user(username="author", workspaces=["workspace"])
+    author = factories.create_airlock_user(username="author", workspaces=["workspace"])
```

Comment:
Am I right that `workspaces=["workspace"]` is now the default so that omitting it (here and elsewhere) would have the same behaviour? Though perhaps it's more readable in the tests to see it explicitly named.

Reply:
Yeah, the test author chose to be explicit, so I let their decision stand